odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -795,6 +829,11 @@ PROCEDURE READCONFIGFILESETTINGS
      PRINT FORMAT("OO-99:",TMPLOOP)+"'"+PARAMOPTOUTINFOTEXT(TMPLOOP)+"'"
      NEWLINE
     END
+   FOR TMPLOOP=1 TO PARAMOPTINOUTINFOTEXTLINES
+    DO
+     PRINT FORMAT("OO-99:",TMPLOOP)+"'"+PARAMOPTINOUTINFOTEXT(TMPLOOP)+"'"
```

Comment:
Wouldn't you want this statement to be "PRINT FORMAT("OIO-99:"........  For PARAMOPTINOUTINFOTEXT(TMPLOOP)

Reply:
Sure enough.  Fixed.  Thanks for catching that.